ddp-automation/src/test/java/org/broadinstitute/ddp/tests/BaseTest.java

```diff
@@ -124,6 +124,7 @@ public static void setupBrowserStackCreds() throws Exception {
             buildName = System.getProperty(BUILD_PROPERTY);
             if (StringUtils.isBlank(buildName)) {
                 buildName = System.getProperty("user.name");
+                logger.info("Build Name: {}", buildName);
```

Comment:
That's a nice touch!

Reply:
I put that there because in browserstack it kept saying the buildname was 'Question Bro'. I still don't know where that came from since test runs showed 'kwestbro'